### PR TITLE
Fix preload warning for invalid as attribute.

### LINKED ISSUE #2492: Warning: `<link rel=preload>` must have a valid `as` value
This is surprising.

Getting this message when loading
http://localhost:8000/max/s/thawing-ridge-70066.herokuapp.com/examples/index.amp.html

Can we always provide a default or make it mandatory?

Lets fix before next dev channel.

### PR DESCRIPTION
**Pull request by mkhatib**
Fixes #2492

### CODE REVIEW

File: src/3p-frame.js [hunk 1]
```diff
@@ -156,7 +156,7 @@ export function addDataAndJsonAttributes_(element, attributes) {
 export function prefetchBootstrap(window) {
   const url = getBootstrapBaseUrl(window);
   const preconnect = preconnectFor(window);
-  preconnect.prefetch(url, 'document');
+  preconnect.prefetch(url);
```
Comment (cramforce) [inline src/3p-frame.js]: Add comment for why the arg was left out.

Comment (mkhatib) [inline src/3p-frame.js]: Done.

**Comment by mkhatib** [comment]
PTAL

**Comment by cramforce** [comment]
LGTM

**Pull request merged**